doc/user/content/free-trial-faqs.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Free trial FAQs"
+description: "Answers to frequency asked questions about Materialize free trials"
+menu:
+  main:
+    parent: "quickstarts"
+    weight: 15
+---
+
+When you [sign up for Materialize](https://materialize.com/register/), you get a free trial account so you can explore the product without worrying about initial costs. This page answers some frequently asked questions about free trials.
+
+## What's included in a free trial?
+
+In Materialize, [cluster replicas](/overview/key-concepts/#cluster-replicas) are the physical resources for doing computational work. Cluster replicas come in various sizes, and each size corresponds to a specific number of [compute credits per hour](https://materialize.com/pricing/).
+
+During your free trial, your rate of credit consumption in a given region cannot exceed 4 credits per hour at any point in time. This limit should accommodate most trial scenarios. If you need additional resources during your trial, please contact sales@materialize.com.
+
+## How long does a free trial last?
+
+14 days.
+
+If you want to continue using Materialize beyond 14 days, please contact sales@materialize.com. Otherwise, Materialize will delete your resources and data at the end of the trial period.
```

Comment:
Same here. It seems unnecessary to mention this twice in a row, too! So maybe you can merge these two questions.

Reply:
I had them merged at one point, but the time limit felt buried in the mix of details around resource limits.